packages/test/test-service-load/src/workloads/gc/gcDataStores.ts

```diff
@@ -999,7 +999,7 @@ export class RootDataObject extends DataObject implements ITestRunner {
 			sendFunc = sendFunc.bind(config.logger);
 			sendFunc(event, logLevel);
 			if (
-				event.eventName.includes("InactiveObject") ||
```

Comment:
I think we should leave this.  The way the timers are now, it will actually transition to Inactive state and set a 0ms timer.  Will "never happen", but no harm in leaving it IMO.  Also, since this line is decoupled from the config where you happen to be setting them to the same value.

Reply:
If you agree you can just commit the suggestion I added on the next line :)

Reply:
Looking at the code, the unreferenced state tracker logic will directly move nodes to tombstone / sweep ready if the timeout has passed. It checks for inactive last.